lib/mutant/cli.rb

```diff
@@ -155,6 +155,9 @@ def add_filter_options(opts)
       opts.on('--ignore-subject EXPRESSION', 'Ignore subjects that match EXPRESSION as prefix') do |pattern|
         add_matcher(:ignore_expressions, config.expression_parser.(pattern))
       end
+      opts.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
```

Comment:
Is revision the best name for this? I've seen you use the branch name, and I assume the SHA works. Does a tag work too? Would the term "reference" be the more generic term for all of these things?